### PR TITLE
test: enable azure blob tests in ci

### PR DESCRIPTION
**Pull request by aryamohanan**
refs INSTA-9955

### CODE REVIEW

**Review by kirrg001 — APPROVED** [review]
LGTM

Just left one question.

File: packages/collector/test/tracing/cloud/azure/blob/test.js [hunk 1]
```diff
@@ -57,6 +54,10 @@ if (!storageAccount || !accountKey) {
   });
 } else {
   mochaSuiteFn('tracing/cloud/azure/blob', function () {
+    const { BlobServiceClient } = require('@azure/storage-blob');
```
[the remark this answers is not in the document]
Comment (aryamohanan) [reply packages/collector/test/tracing/cloud/azure/blob/test.js:57]: Minimum supported version is v18, we don't need to require it if the version is not supported.

[the remark this answers is not in the document]
Comment (kirrg001) [reply packages/collector/test/tracing/cloud/azure/blob/test.js:57]: There are some cases for production code where it makes sense to require a pkg in a function for performance reasons, but this is a test. I'd leave the requires on top for readability.

[the remark this answers is not in the document]
Comment (aryamohanan) [reply packages/collector/test/tracing/cloud/azure/blob/test.js]: Updated.

**Pull request merged**